testing-guidance/workflow-tests/physics/physics-lkg-workflows.md

```diff
@@ -83,10 +83,10 @@ Any project that has the following Gems enabled:
-| PhysX colliders can have their offset and contact offset modified and their collision behavior is impacted accordingly | <ol><li>Set an offset and contact offset for PhysX colliders with all supported [PhysX Collider shape(s)](#physx-collider-shapes).</li><li>Set up a scene in which created PhysX colliders with modified offsets and contact offsets collide with simulated rigid bodies moved by PhysX forces.</li><li>Set a contact offset and translation offset for for each of [PhysX Shape Collider shape(s)](#physx-shape-collider-shapes) by using flag enabled Translation Offset (translation offset can be enabled by adding _"EnableShapeComponentTranslationOffset": true_ under _"Amazon" -> "Preferences"_ in _editorpreferences.setreg_ file).</li><li>Set up a scene in which created PhysX Shape colliders with modified shape translation offsets and contact offsets collide with simulated rigid bodies moved by PhysX forces.</li></ol> | <ul><li>PhysX Colliders can have their offsets and contact offsets modified.</li><li>PhysX Shape colliders can have their contact offsets and flag enabled shape translation offsets modified.</li><li>PhysX colliders and PhysX shape colliders with offsets modified as described above collide with PhysX rigid bodies and other PhysX entities during simulations.</li></ul>                                                     
+| PhysX Materials can be applied to colliders and impact their collisions  | <ol><li>Create a PhysX Primitive Collider, PhysX Mesh Collider and PhysX Shape Collider entity setup with appropriate Collider componenets for each PhysX Material under test.</li><li>Add the desired PhysX Material(s) under test to the entities created.</li><li>Set up the scene so that the entities with PhysX Materials under test will collide with something.</li><li>Run the Simulation in Editor Game Mode or in the Game Launcher.</li></ol>                                                     | <ul><li>PhysX Colliders can have PhysX Materials applied.</li><li>PhysX Material properties are applied to Colliders and impact their collisions.</li>                                                                                                                                                                   |
```

Comment:
```suggestion
| PhysX Materials can be applied to colliders and impact their collisions  | <ol><li>Create a PhysX Primitive Collider, PhysX Mesh Collider and PhysX Shape Collider entity setup with appropriate Collider components for each PhysX Material under test.</li><li>Add the desired PhysX Material(s) under test to the entities created.</li><li>Set up the scene so that the entities with PhysX Materials under test will collide with something.</li><li>Run the Simulation in Editor Game Mode or in the Game Launcher.</li></ol>                                                     | <ul><li>PhysX Colliders can have PhysX Materials applied.</li><li>PhysX Material properties are applied to Colliders and impact their collisions.</li>                                                                                                                                                                   |
```